Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -191,6 +191,17 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         ASSERT(!exception.message.isNull());
         promise.reject(exception.toException());
     };
+
```

Comment:
Let's handle a block to handle creates being cancelled as well similar as the `if (requestOptions.signal) {` block below. The signal is`createOptions.signal.`